app/src/main/res/values/themes.xml

```diff
@@ -1,6 +1,6 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.TeamTrack" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
try to use just one theme
check AndroidManifest to remove all another

Reply:
done